src/components/UserMenu.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <div>
+    <oc-button id="_userMenuButton" class="oc-topbar-personal uk-height-1-1" variation="primary" aria-label="$gettext('User Menu')" ref="menuButton">
+      <avatar class="oc-topbar-personal-avatar" :userid="userId" />
+      <span class="oc-topbar-personal-label">{{ userDisplayName }}</span>
+    </oc-button>
+    <oc-drop toggle="#_userMenuButton" mode="click" :options="{pos:'bottom-right'}" class="uk-width-large" ref="menu">
+      <div class="uk-card-body uk-flex uk-flex-middle uk-flex-column">
+        <avatar
+          :userid="userId"
+          width="128"
+        />
+        <h3 class="uk-card-title">{{ userDisplayName }}</h3>
+        <span v-if="userEmail">{{ userEmail }}</span>
+        <oc-button type="a" href="/account">Manage your account</oc-button>
+        <br/>
+        <oc-button type="a" @click="logout()">Sign out</oc-button>
+      </div>
+      <div class="uk-card-footer uk-flex uk-flex-middle uk-flex-column">
+        <span>Version: {{appVersion.version}}-{{appVersion.hash}} ({{appVersion.buildDate}})</span>
+      </div>
+    </oc-drop>
+  </div>
+</template>
+
+<script>
+import appVersionJson from '../../build/version.json'
+import Avatar from './Avatar.vue'
+
+export default {
+  components: {
+    Avatar
+  },
+  props: {
+    userId: {
+      type: String,
+      required: true
+    },
+    userDisplayName: {
+      type: String,
+      required: true
+    },
+    userEmail: {
+      type: String,
+      required: false,
+      default: null
+    }
+  },
+  data () {
+    return {
+      appVersion: appVersionJson
+    }
+  },
+  watch: {
+    visible (val) {
+      if (val) {
+        this.focusFirstLink()
+      } else {
+        this.$emit('closed')
+      }
+    }
+  },
+  computed: {
+    _logoutItemText () {
+      // return this.$gettextInterpolate(this.$gettext('Exit %{product}'), { product: this.configuration.theme.general.name })
```

Comment:
I guess we need to pass in the product name via props as well - or remove any white labeling ...

Reply:
obsolete, will keep the generic text "Sign out"